File: 1. Two Sum.js
var twoSum = function(nums, target) {
    let fo = [];
    for (let i = 0; i < nums.length; i++) {
        for (let j = i + 1; j < nums.length; j++) {
            if (nums[i] + nums[j] === target) {
                fo.push(i, j);
            }
        }
    }
    return fo;
};

console.log(twoSum([3, 3], 6));  // Output: [0, 1]

File: 1.two_sum.py
def two_sum(nums, target):
    fo = []
    for i in range(len(nums)):
        for j in range(i + 1 , len(nums)):
            if nums[i] + nums[j] == target:
                fo.extend([i, j])
    return fo



# print(two_sum([3, 3], 6))
